addons/mail/static/src/less/chat_window.less

```diff
@@ -95,6 +95,12 @@
 
     .o_chat_composer input {
         width: 100%;
+        outline: 0;
+        height: 39px;
+        padding: 10px;
+        font-size: 13px;
+        border: none;
+        border-top: 1px solid #dee2e6;
```

Comment:
Won't this change the design of the backend chat windows?

Reply:
No it uses a textarea instead of an input

Reply:
So this only styles the frontend chat window? Why is this rule in the backend assets then?

Reply:
Just checked, so here am I in the backend. The one on the right has a textarea in its o_chat_composer. So it's ok, but when you click on the message icon in the systray and you create a new message you have the left one chat which is the same than frontend livechat except that the composer is in display: none. So I think it's ok too

![chatters](https://user-images.githubusercontent.com/9156538/74231148-29fe7900-4cc6-11ea-825a-791a6bccca04.png)